go/oasis-node/integrationrunner/integration_test.go

```diff
@@ -0,0 +1,17 @@
+package integrationrunner
+
+import (
+	"flag"
+	"testing"
+)
+
+var run = flag.Bool("integration.run", false, "Run the node")
+
+func TestIntegration(t *testing.T) {
+	if !*run {
+		t.Skip("Pass -integration.run to run the node")
```

Comment:
This is to avoid running this during normal test runs? Can you add some comments around here (maybe package-level) how this is used?

Reply:
yeah. I'll add a comment